### PR TITLE
chore(eslint): enable comma-dangle always-multiline

### PR DESCRIPTION
**Pull request by jogold**
See [here](https://github.com/aws/aws-cdk/pull/7409/files?file-filters%5B%5D=.js#diff-52f1fdeb95d5b1b5d0a3f1af3a20e3d5) for only relevant change.

image

### Commit Message
chore(eslint): enable comma-dangle always-multiline

This ensures clean diffs.

See https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8
### End Commit Message

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by jogold** [comment]
Need to update the integ test expectations, will do it if OK with the proposed linting rule.

**Comment by jogold** [comment]
@nija-at can you take a look when you have time? Considering the number of changes here, it will quickly become hard to rebase/merge from master if too many commits are added to master in the meantime. Thanks!

File: tools/cdk-build-tools/config/eslintrc.js [hunk 1]
```diff
@@ -32,13 +32,16 @@ module.exports = {
     // Require use of the `import { foo } from 'bar';` form instead of `import foo = require('bar');`
     '@typescript-eslint/no-require-imports': [ 'error' ],
     '@typescript-eslint/indent': [ 'error', 2 ],
+
+    // Style
     'quotes': [ 'error', 'single', { avoidEscape: true } ],
+    'comma-dangle': [ 'error', 'always-multiline' ], // ensures clean diffs
```
Comment (nija-at) [inline tools/cdk-build-tools/config/eslintrc.js:38]: can you add the link to the medium article here as well?

Comment (jogold) [reply tools/cdk-build-tools/config/eslintrc.js]: done, can you reapprove?

**Pull request merged**